### DISCUSSION TITLE
Redlining: wenn Bemaßung mit "Abschnitt schließen/neuen beginnen" beendet wird, ist "Übernehmen/weiteres Zeichnen" nicht möglich.

### DISCUSSION THREAD

**Discussion by faulgurke** [e-netze/webgis-community#207] [Q&A]
Das Bemaßen mehrerer Linien, welche jeweils mit "Abschnitt abschließen" beendet werden führt dazu, dass "Übernehmen / weiteres Zeichnen" nicht reagiert. 
Es ist immer die Bemaßung aktiv, obwohl man beispielsweise "Fläche" aus dem Menü wählt.
Die Bemaßung muss gelöscht werden, damit man andere Zeichenwerkzeuge aktivieren kann. 

Ist es ein Feature, dass jede Bemaßungslinien separat mit "Übernehmen / weiteres Zeichnen" oder Doppelklick beendet werden muss?

**Comment by gis-eni** [comment]
Mit welcher Version arbeitest du? Ich kann das hier nicht nachvollziehen.

**Reply by faulgurke** [reply]
Version 5.24.1801.

**Reply by gis-eni** [reply]
Kommando zurück. Jetzt konnte ich es nachvollziehen. Du beendest nicht mit nur mit "Abschnitt abschließen" sondern zeichnest wirklich ein Multipart Feature. Dann bekommst das oben beschriebene verhalten.

**Reply by faulgurke** [reply]
Genau. Aber den Vorschlag mit der neuen Version werde ich auch befolgen ;-)

**Reply by gis-eni** [reply]
Wart vielleicht noch mit dem Update, bis dieser Fehler behoben ist.

Kommt dann mit nächsten Build. Dort kannst du dann mit Bemaßungslinie und Hectolinie eine Multipart Linien mehr zeichnen. Der Button "Neuer Abschnitt" Fehlt dann im Menü.

Ab Build 5/6.24.4201
